app/assets/javascripts/darkswarm/directives/stripe_elements.js.coffee

```diff
@@ -1,29 +1,49 @@
 Darkswarm.directive "stripeElements", ($injector, StripeElements) ->
   restrict: 'E'
-  template: "<label for='card-element'>\
-             <div id='card-element'></div>\
-             <div id='card-errors' class='error'></div>\
+  template: "<div class='card-element'>\
+             <label for='card-number-element'>\
+             <span>Card number</span>\
+             <div id='card-number-element' class='field'></div>\
+             </label>\
+             <label for='card-expiry-element'>\
+             <span>Expiry date</span>\
+             <div id='card-expiry-element' class='field'></div>\
+             </label>\
+             <label for='card-cvc-element'>\
+             <span>CVC</span>\
+             <div id='card-cvc-element' class='field'></div>\
+             <div id='card-errors' class='error'></div>\              
              </label>"
```

Comment:
Since this is getting more complicated perhaps it should be a separate haml file in the templates/ folder.